dat/missions/neutral/cargo_rush.lua

```diff
@@ -161,10 +164,12 @@ function land()
       if intime then
       -- Semi-random message.
       tk.msg(cargo_land_title, string.format( cargo_land_p2[rnd.rnd(1, #cargo_land_p2)], cargo_land_p1[rnd.rnd(1, #cargo_land_p1)], _(cargo) ))
+      misn.appendLog("Cargo delivered on time to " .. destplanet:name() .. ", payment " .. math.floor(reward) .. " credits")
    else
       -- Semi-random message for being late.
       tk.msg(cargo_land_title, string.format( cargo_land_p3[rnd.rnd(1, #cargo_land_p3)], cargo_land_p1[rnd.rnd(1, #cargo_land_p1)], _(cargo), reward/2, reward ))
       reward = reward / 2
+      misn.appendLog("Cargo delivered late to " .. destplanet:name() .. ", payment " .. math.floor(reward) .. " credits")
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.